clients/src/main/java/org/apache/kafka/clients/admin/internals/AlterConsumerGroupOffsetsHandler.java

```diff
@@ -159,10 +159,11 @@ private void handleError(
         Set<CoordinatorKey> groupsToRetry
     ) {
         switch (error) {
-            // If the coordinator is in the middle of loading, then we just need to retry.
+            // If the coordinator is in the middle of loading, or rebalance is in progress, then we just need to retry.
             case COORDINATOR_LOAD_IN_PROGRESS:
+            case REBALANCE_IN_PROGRESS:
                 log.debug("OffsetCommit request for group id {} failed because the coordinator" +
-                    " is still in the process of loading state. Will retry.", groupId.idValue);
+                    " is still in the process of loading state or the group is rebalancing. Will retry.", groupId.idValue);
```

Comment:
I wonder if it would be preferable to use a more generic message which clearly mention the error encountered: `OffsetCommit request for group id {} returned error {}. Will retry.`. Without mentioning the received error, we don't really know what happened so the log is not that useful.

Reply:
Make sense to me.